stylesheet.css

```diff
@@ -0,0 +1,69 @@
+body {
+    background-color: #f1f1f1;
+    font-family: calibri;
+    color:#5f5f5f;
+}
+
+header {
+  background: #5f5f5f;
+    padding: 1em;
+}
+
+header a {
+    color: white;
+    margin: 2em;
+    text-decoration:none;
+    
+}
+h2 {
+    border-left: 22px;
+    border-left-color: aqua;
+    
+
+}
+header nav {
+     min-height: 1em;
+}
+header a:hover {
+   background:#6d8263;
+}
+
+header a:active{
```

Comment:
same with spaces before {
I would say you can stick to this style: header a:active {